tensorflow/compiler/xla/service/gpu/tests/BUILD

```diff
@@ -594,6 +594,12 @@ glob_lit_tests(
         "no_pip",
     ],
     driver = "@llvm-project//mlir:run_lit.sh",
+    tags_override = {
+        "reduction_vectorization_sm50.hlo": ["no_rocm"],
```

Comment:
Maybe rename those targets as well to make it explicit that they are NVIDIA-specific and test the output PTX?

[Also is there a way to test vectorization on ROCm in the same way?]

Reply:
>[Also is there a way to test vectorization on ROCm in the same way?]

There should be in theory. Let me do some digging and figure it out